src/Compilers/CSharp/Portable/Binder/Binder_Conversions.cs

```diff
@@ -700,7 +700,7 @@ private bool MemberGroupFinalValidationAccessibilityChecks(BoundExpression recei
             }
             else if (WasImplicitReceiver(receiverOpt))
             {
-                if (InFieldInitializer && !ContainingType.IsScriptClass || InConstructorInitializer || InAttributeArgument)
+                if (InFieldInitializer && !ContainingType!.IsScriptClass || InConstructorInitializer || InAttributeArgument)
```

Comment:
>ContainingType [](start = 43, length = 14)

Is the suppression okay because `InFieldInitializer` is true at this point?

Reply:
Correct.

---
In reply to: [367183871](https://github.com/dotnet/roslyn/pull/40946#discussion_r367183871) [](ancestors = 367183871)